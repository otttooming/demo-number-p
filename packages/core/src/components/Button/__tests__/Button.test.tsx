import * as React from "react";
import { shallow, ShallowWrapper } from "enzyme";
import Button, { ButtonProps } from "../Button";

const props: ButtonProps = {};

describe("<Button>", () => {
  let view: ShallowWrapper<ButtonProps>;
  beforeEach(() => {
    view = shallow(<Button {...props} />);
  });

  it("should match snapshot", () => {
    expect(view).toMatchSnapshot();
  });
});
