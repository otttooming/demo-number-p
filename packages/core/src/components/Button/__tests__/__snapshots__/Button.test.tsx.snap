// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Button> should match snapshot 1`] = `
<styled.button
  onClick={[Function]}
>
  <styled.div />
</styled.button>
`;
